pages/data-migration/migrate-from-rdbms-directly.md

```diff
@@ -0,0 +1,279 @@
+and outlined the [benefits of graph
+databases](https://memgraph.com/blog/the-benefits-of-using-a-graph-database-instead-of-sql).
+
+## Prerequisites
+
+To follow along, you will need:
+
+- **Memgraph MAGE** which includes **Memgraph** and **MAGE**, an advanced graph
+  algorithms and modules library. To install the Memgraph MAGE and set it up,
+please follow the [getting started guide](/getting-started).
+- (optional) A running relational database either with your own schema and data
+  or the data we provided to populate the tables. Here we will be using PostgreSQL, but
+  you can also choose whatever external source, if it is supported inside 
+  [MAGE available migration algorithms](/advanced-algorithms/available-algorithms/migrate).
+
+## Data model
+
+Let's first get acquainted with the model. In our Postgres tables, we will be having `Musicians`
+and `Bands`, with their respective properties.
+
+The `Musicians` table will have an id, name and instrument they play:
+```sql
+CREATE TABLE Musicians (
+    musician_id SERIAL PRIMARY KEY,
+    name TEXT NOT NULL,
+    instrument TEXT
+);
+
+INSERT INTO Musicians (name, instrument) VALUES
+('John Doe', 'Guitar'),
+('Jane Smith', 'Vocals'),
+('Alice Johnson', 'Bass'),
+('Bob Lee', 'Drums'),
+('Charlie Brown', 'Keyboard'),
+('Eva Green', 'Guitar');
+```
+
+Bands will have the band id, and the name of the band:
+```sql
+CREATE TABLE Bands (
+    band_id SERIAL PRIMARY KEY,
+    band_name TEXT NOT NULL
+);
+
+INSERT INTO Bands (band_name) VALUES
+('The Rockers'),
+('The Jazz Masters'),
+('The Classical Ensemble'),
+('Pop Sensations');
+```
+
+And they will be connected with a join table:
+```sql
+CREATE TABLE BandMemberships (
+    musician_id INTEGER,
+    band_id INTEGER,
+    PRIMARY KEY (musician_id, band_id),
+    FOREIGN KEY (musician_id) REFERENCES Musicians (musician_id) ON DELETE CASCADE,
+    FOREIGN KEY (band_id) REFERENCES Bands (band_id) ON DELETE CASCADE
+);
+
+INSERT INTO BandMemberships (musician_id, band_id) VALUES
+(1, 1),  -- John Doe is a guitarist in "The Rockers"
+(2, 1),  -- Jane Smith is a vocalist in "The Rockers"
+(3, 1),  -- Alice Johnson is a bassist in "The Rockers"
+(4, 1),  -- Bob Lee is a drummer in "The Rockers"
+(2, 2),  -- Jane Smith is also a vocalist in "The Jazz Masters"
+(5, 2),  -- Charlie Brown plays keyboard in "The Jazz Masters"
+(6, 4),  -- Eva Green plays guitar in "Pop Sensations"
+(2, 4);  -- Jane Smith also sings in "Pop Sensations"
+```
+
+The exercise is to migrate `Bands` and `Musicians` rows as nodes, and the mapping table as
+relationships. The steps below are provided so you can quickly glance at them at any time:
+
+1. Create necessary indices (important for relationship import)
+2. Switch to in-memory analytical mode (to eliminate memory overhead)
+3. Import nodes from corresponding tables
+4. Import relationships from corresponding tables
+5. Switch back to in-memory analytical mode
```

Comment:
Uh, we switched to analytical in step 2. Do you mean switch back to transactional?